# StartEnvironment.srv
string experiment_path
---
bool success
string message
